packages/core/src/application.ts

```diff
@@ -59,10 +70,11 @@ export class Application extends Context {
   protected _bindSequence(): void {
     // TODO(bajtos, ritch, superkhau) figure out how to integrate this single
     // sequence with custom sequences contributed by components
-    const sequence = this.options && this.options.sequence
-      ? this.options.sequence
-      : Sequence;
-    this.bind('sequence').toClass(sequence);
+    const sequence: Constructor<SequenceHandler> =
+      this.options && this.options.sequence
+        ? this.options.sequence
+        : DefaultSequence;
+    this.sequence(sequence);
```

Comment:
I'm wondering if we should remove this complication as discussed before. Since `app.sequence` is working sugar now, can we modify `this.bindSequence` in the ctor to be a bit smarter (ie. move the optional default logic to where it belongs):

```
constructor(public options?: ApplicationOptions) {
  ...
  options && options.sequence
    ? this.sequence(options.sequence)
    : this.sequence(DefaultSequence);
}
```

Then remove _bindSequence altogether. Thoughts?